lib/ansible/modules/system/seport.py

```diff
@@ -156,9 +156,13 @@ def semanage_port_get_type(seport, port, proto):
     :rtype: tuple
     :return: Tuple containing the SELinux type and MLS/MCS level, or None if not found.
     """
-    ports = port.split('-', 1)
-    if len(ports) == 1:
-        ports.extend(ports)
+    if isinstance(port, str):
+        ports = port.split('-', 1)
+        if len(ports) == 1:
+            ports.extend(ports)
+            pass
```

Comment:
`pass` seems useless to me.

Reply:
Is that a thing?  I will remove it, I do it thoughtlessly.  It is literally irrelevant, it compiles out, and it ensures proper formatting during cut and pasting.

However, I know little/nothing about code hygiene in Ansible.